packages/engine/src/scene/systems/ShadowSystem.tsx

```diff
@@ -193,6 +193,7 @@ const EntityCSMReactor = (props: { entity: Entity; rendererEntity: Entity; rende
 
   useEffect(() => {
     if (!csm) return
+    csm.updateFrustums()
```

Comment:
this call and the whole useEffect is redundant now, let's  remove the csm.UpdateFrustums call, comment out the useEffect and add a todo to add support for dynamic cascades